dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -77,4 +77,22 @@
     <changeSet author="nolwarre (generated)" id="notificationsCharLimit">
         <modifyDataType columnName="message" newDataType="varchar(1024)" tableName="notification"/>
     </changeSet>
+    <changeSet author="gluu (generated)" id="1601569350016-13">
+        <createTable tableName="collection_entry_version">
```

Comment:
Can you have a unique constraint on all 3 columns together (entry_id, version_id, collection_id)? The problem is that version_id is nullable; I don't know if that will work.

Maybe this: https://stackoverflow.com/questions/8289100/create-unique-constraint-with-null-columns

Reply:
We did this before to enforce the nullable but unique workflow names

Reply:
Done